core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -339,8 +340,9 @@ private[log] class Cleaner(val id: Int,
         
     // group the segments and clean the groups
     info("Cleaning log %s (discarding tombstones prior to %s)...".format(log.name, new Date(deleteHorizonMs)))
-    for (group <- groupSegmentsBySize(log.logSegments(0, endOffset), log.config.segmentSize, log.config.maxIndexSize))
+    for (group <- groupSegmentsBySize(log.logSegments(0, endOffset), log.config.segmentSize, log.config.maxIndexSize)) {
```

Comment:
No need to {} for single line statements. There are a few other places like that.